Domain_classes/Account.py

```diff
@@ -1,10 +1,14 @@
+
 class Account():
-    def __init__(self, account_id,name,open_transaction,balance):
+    def __init__(self, account_id,name,balance:int, open_transaction=None):
         self.account_id = account_id
         self.name = name
         self.open_transaction = open_transaction
         self.balance = balance
 
+    def __str__(self):
+        return f" numer konta:{self.account_id}, imie:{self.name}, saldo:{self.balance} złotych"
```

Comment:
Super pomysł z tym nadpisaniem metody str

Reply:
na samym poczatku to napisalem, kiedy sie zagalopowalem z pisaniem bardziej tej calej logiki zamiast testow, probowalem napisac ten generator slownika kont i wkurzalo mnie ze ciagle wypisywal mi object class Account 123553 itp zamiast to co tam jest wiec dopisalem ta metode ;)